package.json

```diff
@@ -42,6 +42,7 @@
     "url": "git://github.com/ampersandjs/ampersand-form-view"
   },
   "scripts": {
+    "validate": "jshint .",
```

Comment:
Oops, I meant to add something about that to my last comment but I must've deleted it on accident. :grin: So yes, I think changing it to `lint` would be the best solution.

Sorry for the nitpicks, this is great and very helpful PR! Thank you! :smile: